### PR TITLE
update mur adr categories

### PR DESCRIPTION
**Pull request by pkfec**
## Summary 

1. Add/Update `case_doc_category_id` field with the latest MUR/ADR document categories to API (`New Doc Order` column in the screenshot below)
Table with latest mur/adr categories:
Screen Shot 2023-02-21 at 6 57 14 PM

2. Change `type` and `mur_type` fields types from text to dropdown list

- Resolves #https://github.com/fecgov/openFEC/issues/5339
- Related PR #https://github.com/fecgov/openFEC/pull/5179

### Required reviewers

2 or more developers

## Impacted areas of the application

-  /legal/search/ API endpoint

## Screenshots

**In Prod:**
Screen Shot 2023-02-21 at 10 13 02 PM

Screen Shot 2023-02-21 at 10 13 19 PM

Screen Shot 2023-02-21 at 10 13 36 PM

**Current branch:**
Screen Shot 2023-02-21 at 10 08 26 PM

Screen Shot 2023-02-21 at 11 44 07 AM

Screen Shot 2023-02-21 at 11 44 45 AM

## How to test
 - Choose Method#1 **OR** Method#2 to test this PR

**Method#1:**  local workspace
- Follow wiki instructions and setup [ElasticSearch7.4](https://github.com/fecgov/openFEC/wiki/Elasticsearch-7.x.0-setup-and-test-locally) on local environment
- Checkout feature branch: `git checkout feature/5339-update-mur-adr-categories`
- run: `pytest`
- Create index `docs` index  on local ES, run: `python cli.py create_index`
- Upload 4 of each mur, adr and admin fine documents onto  local ES service

```
python cli.py load_current_murs 4908
python cli.py load_current_murs 5545
python cli.py load_current_murs 6621
python cli.py load_current_murs 7792

python cli.py load_adrs 218
python cli.py load_adrs 243
python cli.py load_adrs 480
python cli.py lload_adrs 999

python cli.py load_admin_fines 2431
python cli.py load_admin_fines 3314
python cli.py load_admin_fines 3757 
python cli.py load_admin_fines 4469
```

- Test filter by `case_doc_category_id` for mur,adr and af case types in legal/search endpoint

**NOTE**: Ref `doc_order` table changes here: https://docs.google.com/spreadsheets/d/1l3YKu-oBzOwBHH986YdyiYaSJTFdJBNxitZkt_Ew4zA/edit#gid=245596029

- Some test urls

**MURs:**
```
- http://127.0.0.1:5000/v1/legal/search/?type=murs (Result set:  "total_murs": 4)

(one case_doc_category_id)
- http://127.0.0.1:5000/v1/legal/search/?type=murs&case_doc_category_id=6 (Result set : "total_murs": 3 )

(more than one case_doc_category_id)
- http://127.0.0.1:5000/v1/legal/search/?type=murs&case_doc_category_id=1&case_doc_category_id=3&case_doc_category_id=5 (Result set: "total_murs": 4)
```

**ADRs:**
```
http://127.0.0.1:5000/v1/legal/search/?type=adrs (returns 4 results)

(one case_doc_category_id)
- http://127.0.0.1:5000/v1/legal/search/?type=adrs&case_doc_category_id=1006 (Result set:  "total_adrs": 2)
- http://127.0.0.1:5000/v1/legal/search/?type=adrs&case_doc_category_id=1002 (Result set:  "total_adrs": 4)

(more than one case_doc_category_id)
- http://127.0.0.1:5000/v1/legal/search/?type=adrs&case_doc_category_id=1006&case_doc_category_id=1005
(Result set:  "total_adrs": 2)
```

**Admin_Fines:**
```
- http://127.0.0.1:5000/v1/legal/search/?type=admin_fines (Result set: "total_admin_fines": 4)
- http://127.0.0.1:5000/v1/legal/search/?type=admin_fines&case_doc_category_id=2001 (Result set: "total_admin_fines": 4)
```

**Method#2**: Deploy this feature branch to `dev` space and test the above URLs

### CODE REVIEW

**Comment by fec-jli** [comment]
Good work.
1)For `type` description, it should be **Legal Document type**, not **case document type**.

2)For ADR test. This url returns 2 records not 1:
http://127.0.0.1:5000/v1/legal/search/?type=adrs&case_doc_category_id=1006

File: webservices/args.py [hunk 1]
```diff
@@ -250,10 +260,13 @@ def make_seek_args(field=fields.Int, description=None):
...
-    'case_doc_category_id': fields.List(IStr(validate=validate.OneOf(['1', '2', '3', '4', '5', '6'])),
-                                    description=docs.CASE_DOCUMENT_CATEGORY_DESCRIPTION),
+    'case_doc_category_id': fields.List(IStr(
+        validate=validate.OneOf(
+            ['1', '2', '3', '4', '5', '6', '1001', '1002', '1003', '1004', '1005', '1006', '2001'])),
+        description=docs.CASE_DOCUMENT_CATEGORY_DESCRIPTION),
 
-    'mur_type': fields.Str(required=False, description=docs.MUR_TYPE),
+    'mur_type': fields.Str(
+        required=False, validate=validate.OneOf(["archived", "current"]), description="Choose a mur type"),
```
Comment (fec-jli) [inline webservices/args.py:269]: Can you update string for mur_ype in docs.py? Thanks

File: webservices/args.py [hunk 2]
```diff
@@ -217,21 +222,26 @@ def make_seek_args(field=fields.Int, description=None):
     'q': fields.Str(required=False, description=docs.TEXT_SEARCH),
     'from_hit': fields.Int(required=False, description=docs.FROM_HIT),
     'hits_returned': fields.Int(required=False, description=docs.HITS_RETURNED),
-    'type': fields.Str(required=False, description=docs.LEGAL_DOC_TYPE),
+    'type': fields.Str(
+            validate=validate.OneOf(["admin_fines", "adrs", "advisory_opinions",
+                                     "murs", "regulations", "statutes"]),
+            description="Choose a case type"),
```
[the remark this answers is not in the document]
Comment (pkfec) [reply webservices/args.py]: @fec-jli Done, can you do a `git pull` and verify the changes?

**Comment by pkfec** [comment]
> Good work. 1)For `type` description, it should be **Legal Document type**, not **case document type**.
> 
> 2)For ADR test. This url returns 2 records not 1: http://127.0.0.1:5000/v1/legal/search/?type=adrs&case_doc_category_id=1006

#1 -  I force pushed the changes, do a `git pull` and verify the changes on more time. 
#2 -  I updated the count

Thanks for your detailed review @fec-jli

**Review by fec-jli — APPROVED** [review]
Great. work as expected. Thanks

**Review by tmpayton — APPROVED** [review]
Great job!

**Pull request merged**